frontend/hub/common/adapters/hubErrorAdapter.tsx

```diff
@@ -46,7 +49,10 @@ export const hubErrorAdapter: ErrorAdapter = (error): ErrorOutput => {
       // Handling Galaxy errors
       for (const e of data.errors as GalaxyError[]) {
         if (e.source && e.source.parameter) {
-          fieldErrors.push({ name: e.source.parameter, message: e.detail || e.title });
+          if (mappedKeys) {
```

Comment:
I might be misreading this, `base_path` is not a field on the form, should we render that one as form level error, since react-hook-from will not be able to associate `base_path` with any present field?

Reply:
In this case the error is associated with the wrong entered `name`, the problem is that the API error returns parameter as a `base_path` instead, so I think this is exactly case for form level error.

(without this users doesn't really know what field is base_path and has to try again)

In this case, the error is associated with the incorrectly entered `name`. The problem is that the API error returns the parameter as a `base_path` instead. So, I think this really is form level error. Without this information, users don't really know which field is base_path and have to try again.